### ISSUE TITLE
create logging features for themoviedb package

### ISSUE THREAD

**Issue by lsmoura** [lsmoura/pomegranate#2]
Just like the one created on the sabnzbd package. Remember to create the proper interface inside the themoviedb package, as we don't want these packages to depend on each other.

**Comment by krishnaindani** [comment]
I would like to contribute on this, is this replicating the logger interface?

**Comment by lsmoura** [comment]
Not really. I just want to have the packages to use a consistent log function until I’m ready to add features to it. Like logging to json or sending log directly to flutter or logdna. 

If you take a look at the sabnzbd package on this same repository, you’ll see how it should be done.

**Comment by krishnaindani** [comment]
Sounds good, will take a look at it

**Comment by lsmoura** [comment]
> hey, I have started looking into this. Could you assign it to me?
> 
> Looking over the codebase one thing I see is this method here https://github.com/lsmoura/pomegranate/blob/main/themoviedb/helper.go#L13 is calling the function https://github.com/lsmoura/pomegranate/blob/main/themoviedb/helper.go#L13 defines. So if I add the logger to the struct and initialize in the new object the function definition will be changing to both. Is this the way we would like to approach or do you have any better way in mind?

I believe you wanted to link line 7? If that’s the case, that’s correct. You’ll also need to create the logging interface on this package (you can copy it from the sabnzbd package).

then, instead of calling the “log.Println” function on this package, you’d replace it with the new logger struct and function.

**Comment by lsmoura** [comment]
Yes! All correct!
